charts/sysdig/CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 ## Change Log
 
 This file documents all notable changes to Sysdig Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
+
+## v1.15.44
+### Minor changes
+* RuntimeScanner:
+    * bump to 1.2.12 version
+    * add support Arm64 (to deploy, benchmark must be disabled and arm64 affinity must be added)
```

Comment:
Can we add this to the README.md documentation? I would say it is a "major" change BTW :)